Source/RapyutaSimulationPlugins/Private/Sensors/ROS2CameraActor.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020-2021 Rapyuta Robotics Co., Ltd.
+
+#include "Sensors/ROS2CameraActor.h"
+
+AROS2CameraActor::AROS2CameraActor() : Super()
+{
+	SceneComponent = CreateDefaultSubobject<USceneComponent>(TEXT("SceneComponent"));
+
+	// Make the scene component the root component
+	RootComponent = SceneComponent;
+	
+	// Setup camera defaults
+	CameraComponent = CreateDefaultSubobject<UROS2CameraComponent>(TEXT("CameraComponent"));

```

Comment:
Since `UCameraComponent` itself is already of `USceneComponent`, maybe it could become the root, so we would not need to create an extra scene component for it I suppose.